components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/config/model/graph/JsGraphBuilder.java

```diff
@@ -150,8 +160,7 @@ public JsGraphBuilder createWith(String script) {
                     (PromptExecutor) this::addShowPrompt);
             globalBindings.put(FrameworkConstants.JSAttributes.JS_FUNC_LOAD_FUNC_LIB,
                     (LoadExecutor) this::loadLocalLibrary);
-            engineBindings.put("exit", (RestrictedFunction) this::exitFunction);
-            engineBindings.put("quit", (RestrictedFunction) this::quitFunction);
```

Comment:
these functions may not need anymore (exitFunction, quitFunction)

Reply:
deprecated